### PR TITLE
Make the console timeline navigation button more noticeable

### PR DESCRIPTION
**Pull request by jaril**
This patch does a few things:
1) It makes the previously subtle seek button in the timeline more prominent by replacing it with a big blue button that pops up from the left when a user hovers on a message. It's now very hard to miss.
2) It adds a label to the seek button when the user hovers on the button (e.g. rewind, fast-forward).
3) Once a user seeks to a message, we leave a purple paused button displayed next to the message they're paused on. This indicates that the user is paused there, and more importantly, is clickable to look at that source line in the debugger.

Demo: https://www.loom.com/share/4a5601aadeee4a46913b8789e6b557b0

Screenshot:
image

### CODE REVIEW

File: src/devtools/client/themes/webconsole.css [hunk 1]
```diff
@@ -994,3 +980,112 @@ a.learn-more-link.webconsole-learn-more-link {
...
+  margin-left: calc(0px - var(--message-left-padding));
+}
+
+.message:hover .button .img {
+  opacity: 100%;
+}
+
+.overlay-container:hover .button {
+  background: linear-gradient(152.2deg, #67DAFF 6.75%, #4382FF 93.14%);
+  box-shadow: 1px 0px 3px rgb(0,0,0,0.3);
```
Comment (gabrielluong) [inline src/devtools/client/themes/webconsole.css:1077]: nit: let's add a space after each comma in rgb()

File: src/ui/components/Timeline/index.js [hunk 2]
```diff
@@ -255,7 +255,8 @@ export class Timeline extends Component {
   };
 
   onPlayerMouseUp = e => {
-    const { hoverTime, hoveringOverMarker } = this.props;
+    const { hoverTime } = this.props;
+    const { hoveringOverMarker } = this.state;
```
Comment (gabrielluong) [inline src/ui/components/Timeline/index.js:259]: It would be nice to add this to the default state in the constructor()

Comment (jasonLaster) [reply src/ui/components/Timeline/index.js:259]: Good catch.

**Review by jasonLaster — APPROVED** [review]
Pushing a couple of visual tweaks

**Pull request merged**